packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -146,10 +149,203 @@ class WidgetInspectorService {
 
   List<String> _pubRootDirectories;
 
+  /// Registers a service extension method with the given name (full
+  /// name "ext.flutter.inspector.name").
+  ///
+  /// The given callback is called when the extension method is called. The
+  /// callback must return a value that can be converted to JSON using
+  /// `json.encode()` (see [JsonEncoder]). The return value is stored as a
+  /// property named `result` in the JSON. In case of failure, the failure is
+  /// reported to the remote caller and is dumped to the logs.
+  @visibleForTesting
+  void registerServiceExtension({
```

Comment:
maybe this class should now just be a Binding?

Reply:
I guess not, it's not its own library, it's tightly part of "widgets".

Reply:
Yes it is tightly coupled with widgets although making it part of binding is very tempting as there are a lot of similarities and some duplicated code.